src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -132,8 +144,7 @@ export const styles = StyleSheet.create({
   },
   gap: {
     height: 1,
-    backgroundColor: '#E0E0E0',
-    marginTop: 50
+    backgroundColor: '#E0E0E0'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `marginTop` property has been removed from the `gap` style. Ensure this doesn't affect the layout negatively.



<!-- This is an auto-generated comment by OSS CodeRabbit -->